R/CDPRemote.R

```diff
@@ -15,7 +15,7 @@ NULL
 #' remote <- CDPRemote$new(host = "localhost", debug_port = 9222, secure = FALSE,
 #'                         local = FALSE, retry_delay = 0.2, max_attempts = 15L)
 #'
-#' remote$connect(callback = NULL)
+#' remote$connect(callback = NULL,)
```

Comment:
the `,` is really necessary here ?
I never use a comma when nothing follow usually... 🤔

Reply:
this is a typo

Reply:
fixed in https://github.com/RLesur/crrri/commit/936f4bc7e29acbf9820a824f70f43bd1d56f6724

Reply:
So my review is useful ! yeah ! 🎉 😉